cmd/wallet-js-sdk/src/gnap/client.js

```diff
@@ -44,6 +45,8 @@ export class Client {
     const gnapResp = await axios.post(url, req, {
       headers: {
         "Content-Type": CONTENT_TYPE,
+        "@target-uri": url,
```

Comment:
target uri doesn't need to be a header - the http-signature algorithm is meant to grab the target URI of the request itself.